hi-ml-cpath/requirements_run.txt

```diff
@@ -17,5 +17,6 @@ tifffile==2022.10.10
 timm==0.6.12
 torch==1.11
 torchmetrics==0.10.2
+torchstain==1.3.0
```

Comment:
What is this library?

Reply:
This was a library that was missing in the cpath run requirements and was needed to make the PR build pass.